### PR TITLE
use lowercase seldon-puid in batch processor

### PR DESCRIPTION
**Pull request by RafalSkolasinski**
fixes grpc with batch processor

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rafalskolasinski**
You can assign the PR to them by writing `/assign @rafalskolasinski` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: testing/scripts/test_batch_processor.py [hunk 1]
```diff
@@ -64,6 +64,42 @@ def test_batch_worker(self, namespace):
...
+            for line in f:
+                count += 1
+                output = json.loads(line)
+                # Ensure all requests are successful
+                assert output.get("data", {}).get("ndarray", False)
+            assert count == batch_size
+
+        logging.info("Sending first batch (grpc): mini-batch size=1")
+
+        start_multithreaded_batch_worker(
```
Comment (majolo) [inline testing/scripts/test_batch_processor.py:78]: This function call is getting really ugly imo, way too many arguments and not clear what each one is doing from here. In Go I would suggest we could refactor and pass a struct `BatchWorkerOpts`, but not sure what the most idiomatic Python approach is. (also this is future work not for this PR)

**Review by majolo — APPROVED** [review]
nice lgtm

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[majolo](https://github.com/SeldonIO/seldon-core/pull/3791#pullrequestreview-825340745 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [majolo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by RafalSkolasinski** [comment]
@axsaucedo I did run batch processor integration tests locally so we do not need to wait for CI in principle

**Pull request merged**